src/paperless_mail/admin.py

```diff
@@ -1,9 +1,25 @@
 from django.contrib import admin
+from django import forms
 from paperless_mail.models import MailAccount, MailRule
 
 from django.utils.translation import gettext_lazy as _
 
 
+class MailAccountAdminForm(forms.ModelForm):
+
+    """Metadata classes used by Django admin to display the form."""
```

Comment:
This seems a bit redundant with almost the same comment below.

Reply:
There might be need for a docstring on both the outer and inner class, though I'm not sure which one should carry more detail.